docs/guides/operator/basic-deployment.adoc

```diff
@@ -232,7 +234,12 @@ Apply the changes:
 ----
 kubectl apply -f example-kc.yaml
 ----
-You can provide an alternative ingress resource pointing to the service `<keycloak-cr-name>-service`.
+You can then provide an alternative ingress resource pointing to the service `<keycloak-cr-name>-service`. For example on OpenShift you are not allowed to use wildcard certificates on passthrough Routes with HTTP/2 enabled. Keycloak on OpenShift with TLS enabled using a wildcard certificate with the default IngressClass will create such a Route. In this case you must disable KeyCloak's built-in ingress with `.spec.ingress.enabled: false`. Access may then be provided by creating a reencrypt Route instead:
```

Comment:
```suggestion
You can then provide an alternative ingress resource pointing to the service `<keycloak-cr-name>-service`. For example on OpenShift you are not allowed to use wildcard certificates on passthrough Routes with HTTP/2 enabled. {project_name} on OpenShift with TLS enabled using a wildcard certificate with the default IngressClass will create such a Route. In this case you must disable {project_name}'s built-in ingress with `.spec.ingress.enabled: false`. Access may then be provided by creating a reencrypt Route instead:
```

Reply:
I think my original wording was ambiguious. In each case it would probably be better to say "The Keycloak CR". WDYT @andymunro @ahus1 ?

Reply:
I'm OK with "A Keycloak CR on OpenShift with ..." and "In this case you must disable the built-in ingress ..." (I think repeating 'Keycloak CR' in the second sentence can be avoided.

Reply:
Ok, applied the change from @andymunro and updated the wording here.